code/modules/integrated_electronics/passive/power.dm

```diff
@@ -94,7 +94,7 @@
 /obj/item/integrated_circuit/passive/power/relay
 	name = "tesla power relay"
 	desc = "A seemingly enigmatic device which connects to nearby APCs wirelessly and draws power from them."
-	w_class = WEIGHT_CLASS_NORMAL
+	w_class = WEIGHT_CLASS_SMALL
 	extended_desc = "The siphon generates 250W of energy, so long as an APC is in the same room, with a cell that has energy.  It will always drain \
```

Comment:
```suggestion
	extended_desc = "The siphon draws 250W of electrical energy, as long as an APC is in the same room, with a cell that has energy. It will always drain \
```